values.yaml

```diff
@@ -97,19 +97,21 @@ postgresql:
   ## postgresqlPort port to connect to postgresql
   # postgresqlPort: 5432
 
-  ## postgresqlUsername is the initial user added to the system. This is freqently the
-  ## postgres user. This user will have superuser abilities.
-  # postgresqlUsername: postgres
-
-  ## postgresqlPassword the password for postgresqlUsername, if using auto generated
-  ## password make sure to save the password as described in the notes. When upgrading
-  ## you will need to provide this secret otherwise a new secret will be generated but
-  ## not saved in the database.
-  postgresqlPassword: supersecret
-
-  # need more space
-  persistence:
-    size: 20Gi
+  auth:
+    ## enablePostgresUser adds the postgres user This user will have superuser abilities.
+    # enablePostgresUser: true
+
+    ## postgresPassword the password for postgres, if using auto generated
+    ## password make sure to save the password as described in the notes. When upgrading
+    ## you will need to provide this secret otherwise a new secret will be generated but
+    ## not saved in the database.
+    postgresPassword: supersecret
```

Comment:
Does this need a matching update of the references to `postgresql.postgresqlPassword` in the README?